# 推杆 0停止 1伸出 2缩回
uint8 push_rod
# 平台 1上升 2上升停止 3下降 4下降停止
uint8 platform
# 传送带 电压0-10V，建议3V以内
float64 conveyor
# 割刀 电压0-10V，建议3V以内
float64 knife
# 前端输送电机 电压0-10V，建议3V以内
float64 shake
# 仿形 0禁用，1使用
uint8 imitate